server/src/models/GameState.ts

```diff
@@ -17,6 +20,11 @@ export class GameState {
         this.heroes[id] = new Hero(team)
     }
 
+    createHouses(id: string) {
+        const team = Object.keys(this.houses).length % 2 === 0 ? 'Human' : 'Zombie'
+        this.houses[id] = new House(team)
+    }
```

Comment:
Any reason this is random?

Reply:
Not random. Mod 2 so odds are zombies and evens are humans.

Reply:
Oh derp. OK. We will just want this to be dependent on the map when houses are specced in the map.